api/src/components/coauthor/controller.ts

```diff
@@ -253,13 +253,19 @@ export const updateConfirmation = async (
             });
         }
 
-        // Is the publication in draft?
+        // Is the publication in locked mode?
         if (publication.currentStatus === 'LIVE') {
             return response.json(403, {
                 message: 'This publication is LIVE and therefore cannot be edited.'
             });
         }
 
+        if (publication.currentStatus === 'DRAFT') {
```

Comment:
This if is not needed. We only need to know if the currentStatus !== 'LOCKED', and return the error message based on the current status: LIVE or DRAFT

Reply:
minor eslint issue
<img width="718" alt="image" src="https://user-images.githubusercontent.com/48569671/221802471-c5d185c9-6299-4305-8196-ba3107484569.png">